src/algorithms/pid/IrtCherenkovParticleID.cc

```diff
@@ -0,0 +1,398 @@
+// Copyright 2023, Christopher Dilks, adapted from Alexander Kiselev's Juggler implementation `IRTAlgorithm`
+// Subject to the terms in the LICENSE file found in the top-level directory.
+
+#include "IrtCherenkovParticleID.h"
+
```

Comment:
`MergeTracksPID` has unit tests, `IrtCherenkovParticleID` does not. Neither are called by default `eicrecon` execution. I can't think of a unit test for `IrtCherenkovParticleID` that would be stable against changes in geometry.

Reply:
I assumed, this could be tested by providing some mock geometry. Otherwise, it might be difficult. I haven't figured out how we could mock geometry to write cases for code that uses DD4hep.

Reply:
Mock DD4hep geometry would be nice. It might be more effort than worth it though, if we can trust the downstream benchmarks enough, at least in this case. Unit testing should be added to `irt` itself, since `IrtCherenkovParticleID` here is effectively a wrapper of `irt`.

Reply:
I'm not the `irt` maintainer, but at least we can dream:
https://github.com/eic/irt/issues/37